components/collectives/sections/LearnMore.js

```diff
@@ -235,7 +235,7 @@ const LearnMore = ({ page }) => {
             <DonateButtonBG alt="" />
             <DonateButtonBGHover alt="" />
             <Link href="/opencollective/donate">
-              <DonateButton buttonStyle="dark" minWidth={'97'}>
+              <DonateButton buttonStyle="marketing" minWidth={'97'}>
```

Comment:
I don't think this `minWidth` is working, you need to either do:

```suggestion
              <DonateButton buttonStyle="marketing" minWidth="97px">
```

Or

```suggestion
              <DonateButton buttonStyle="marketing" minWidth={97}>
```